src/main/java/uk/gov/hmcts/reform/sendletter/services/LetterService.java

```diff
@@ -78,9 +79,13 @@ public UUID send(Letter letter, String serviceName) throws JsonProcessingExcepti
         DbLetter dbLetter = new DbLetter(id, serviceName, letter);
 
         //Save message details to db for reporting
-        letterRepository.save(dbLetter, Instant.now(), messageId);
-        placeLetterInQueue(dbLetter, messageId);
-        return id;
+        SendLetterResponse response = letterRepository.save(dbLetter, Instant.now(), messageId);
```

Comment:
I can see that the repository returns this bigger object so that the service can figure out what to do. I'd make it a separate class, though, as we SendLetterResponse doesn't need this `updated` field.